.github/workflows/uploadBinaries.yml

```diff
@@ -99,23 +99,42 @@ jobs:
             -DLLVM_FORCE_ENABLE_STATS=${{ matrix.build_config.stats }} \
             -DCIRCT_RELEASE_TAG_ENABLED=ON \
             -DCIRCT_RELEASE_TAG=${{ github.ref_name }} \
-            -DCMAKE_EXPORT_COMPILE_COMMANDS=OFF
+            -DCMAKE_EXPORT_COMPILE_COMMANDS=OFF \
+            -DCMAKE_INSTALL_PREFIX=`pwd`/../install
           ninja
           ninja check-circt check-circt-unit
+          ninja install
+          cd ..
+
       - name: Display Files
-        run: file build/bin/*
+        run: |
+          file install/*
```

Comment:
Does `tree -C install/` work? :innocent: .  But all good as-is.

Reply:
I sort of just copied this from the previous thing, I'm not sure that I actually care to list the files, I'd rather run `ldd` and `otool -L`. Also my Mac doesn't have `tree` by default so probably no 😬

Reply:
Haha, makes sense and agreed.  Thanks!